Unicode/is_Ligature.c

```diff
@@ -1,104 +1,21 @@
 /*
 Copyright: 2012 Barry Schwartz
+    Create is_Ligature.c (using a python script) to test for Vulgar fractions
 Copyright: 2016 Joe Da Silva
-License: BSD-3-clause
+    Re-write is_Ligature.c to test for ligatures, Vulgar and Other-fractions.
+Copyright: 2016 Gioele Barabucci
+    Simplify makeutype.c code. Split is_Ligature.c, create is_Ligature_data.h
+License: GPL-3+
```

Comment:
@gioele, it's been a long time since I've looked at it, but RedHat has/had a contribution guide, and one of the things that stuck in my mind was, smallish patches of about 10ish lines of code was considered a contribution, and large edits needed a copyright. Although there are many patches in this list you originally created, the sum total has some structural changes of significance in relation to the size of the file and what they do.
There are 2 somewhat recent Debian issues (prior to pulling in this new post-20120731-FontForge) which is also asking about clarifying copyrights (example Unicode/charset/* ), so you'll likely see more of these headers added after examining file histories.
I don't think renaming variables, editing space/formatting, or shuffling functions count, and there is a lot of gray area in this too.